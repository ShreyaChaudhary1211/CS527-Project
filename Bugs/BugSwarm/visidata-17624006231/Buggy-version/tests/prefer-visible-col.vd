sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	Units		addcol-expr	Units	=	create new column from Python expression, with column names as variables
sample	Units		freeze-col		'	add a frozen copy of current column with all cells evaluated
sample	Units		hide-col		-	Hide current column
sample	Units		hide-col		-	Hide current column
sample	Units_frozen		rename-col	Units	^	edit name of current column
sample			select-rows		gs	select all rows
sample	Units	0	setcol-input	0	ge	set contents of current column for selected rows to same input
sample			unselect-rows		gu	unselect all rows
sample	Units		type-int		#	set type of current column to int
sample	Units		addcol-expr	Units + 1	=	create new column from Python expression, with column names as variables
