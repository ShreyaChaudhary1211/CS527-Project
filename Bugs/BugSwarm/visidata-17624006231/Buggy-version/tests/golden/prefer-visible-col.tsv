OrderDate	Region	Rep	Item	Units	Units + 1	Unit_Cost	Total
2016-01-06	East	Jones	Pencil	0	1	1.99	189.05
2016-01-23	Central	Kivell	Binder	0	1	19.99	999.50
2016-02-09	Central	Jardine	Pencil	0	1	4.99	179.64
2016-02-26	Central	Gill	Pen	0	1	19.99	539.73
2016-03-15	West	Sorvino	Pencil	0	1	2.99	167.44
2016-04-01	East	Jones	Binder	0	1	4.99	299.40
2016-04-18	Central	Andrews	Pencil	0	1	1.99	149.25
2016-05-05	Central	Jardine	Pencil	0	1	4.99	449.10
2016-05-22	West	Thompson	Pencil	0	1	1.99	63.68
2016-06-08	East	Jones	Binder	0	1	8.99	539.40
2016-06-25	Central	Morgan	Pencil	0	1	4.99	449.10
2016-07-12	East	Howard	Binder	0	1	1.99	57.71
2016-07-29	East	Parent	Binder	0	1	19.99	1619.19
2016-08-15	East	Jones	Pencil	0	1	4.99	174.65
2016-09-01	Central	Smith	Desk	0	1	125.00	250.00
2016-09-18	East	Jones	Pen Set	0	1	15.99	255.84
2016-10-05	Central	Morgan	Binder	0	1	8.99	251.72
2016-10-22	East	Jones	Pen	0	1	8.99	575.36
2016-11-08	East	Parent	Pen	0	1	19.99	299.85
2016-11-25	Central	Kivell	Pen Set	0	1	4.99	479.04
2016-12-12	Central	Smith	Pencil	0	1	1.29	86.43
2016-12-29	East	Parent	Pen Set	0	1	15.99	1183.26
2017-01-15	Central	Gill	Binder	0	1	8.99	413.54
2017-02-01	Central	Smith	Binder	0	1	15.00	1305.00
2017-02-18	East	Jones	Binder	0	1	4.99	19.96
2017-03-07	West	Sorvino	Binder	0	1	19.99	139.93
2017-03-24	Central	Jardine	Pen Set	0	1	4.99	249.50
2017-04-10	Central	Andrews	Pencil	0	1	1.99	131.34
2017-04-27	East	Howard	Pen	0	1	4.99	479.04
2017-05-14	Central	Gill	Pencil	0	1	1.29	68.37
2017-05-31	Central	Gill	Binder	0	1	8.99	719.20
2017-06-17	Central	Kivell	Desk	0	1	125.00	625.00
2017-07-04	East	Jones	Pen Set	0	1	4.99	309.38
2017-07-21	Central	Morgan	Pen Set	0	1	12.49	686.95
2017-08-07	Central	Kivell	Pen Set	0	1	23.95	1005.90
2017-08-24	West	Sorvino	Desk	0	1	275.00	825.00
2017-09-10	Central	Gill	Pencil	0	1	1.29	9.03
2017-09-27	West	Sorvino	Pen	0	1	1.99	151.24
2017-10-14	West	Thompson	Binder	0	1	19.99	1139.43
2017-10-31	Central	Andrews	Pencil	0	1	1.29	18.06
2017-11-17	Central	Jardine	Binder	0	1	4.99	54.89
2017-12-04	Central	Jardine	Binder	0	1	19.99	1879.06
2017-12-21	Central	Andrews	Binder	0	1	4.99	139.72
